src/libcore/ffi.rs

```diff
@@ -268,35 +268,86 @@ mod sealed_trait {
                reason = "the `c_variadic` feature has not been properly tested on \
                          all supported platforms",
                issue = "44930")]
-    pub trait VaArgSafe {}
+    pub trait VaArgSafe {
```

Comment:
Maybe `VaArgSafe` should be renamed to something more appropriate, like `VaArgType`.